src/javascript/tracking.js

```diff
@@ -145,10 +145,10 @@ Tracking.prototype.init = function(config) {
 
 	settings.set('page_sent', false);
 
-	const cookieDomain = config ? config.cookieDomain : false;
+	const cookieDomain = config ? config.cookieDomain : '';
 
 	// Set up the user from stored - may later be updated by config
-	user.init(false, cookieDomain);
+	user.init('', cookieDomain);
```

Comment:
`User.init` takes two strings, if the strings are empty it triggers the function to set some default value.